sample-resource-plugin/src/integrationTest/java/org/opensearch/sample/AbstractSampleResourcePluginTests.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sample;
+
+import org.opensearch.security.spi.resources.ResourceAccessScope;
+import org.opensearch.test.framework.TestSecurityConfig;
+
+import static org.opensearch.sample.utils.Constants.RESOURCE_INDEX_NAME;
+import static org.opensearch.sample.utils.Constants.SAMPLE_RESOURCE_PLUGIN_PREFIX;
+import static org.opensearch.security.dlic.rest.support.Utils.PLUGIN_RESOURCE_ROUTE_PREFIX;
```

Comment:
It might be more a personal taste and nitpicking but I'd recommend to avoid using constants from the code base to be tested in the code base that executes the tests.

The reason:

Suppose I accidentially add a random a character into the value of `PLUGIN_RESOURCE_ROUTE_PREFIX` .. this would break all plugins and clients which use this path. However, the test would happily pick up the changes and continue being green.

Using values/constants that are separate from the main code base, however, would allow us to notice such accidential changes, as the tests would fail then.

Reply:
This is no longer used.